source/vir/src/prelude.rs

```diff
@@ -275,6 +277,10 @@ pub(crate) fn prelude_nodes() -> Vec<Node> {
             (<= 0 ([height] x))
             :pattern (([height] x))
         )))
+
+        // BitVector
+        // these are the uninterpreted integer versions
+        (declare-fun [uint_xor] (Int Int) Int)
```

Comment:
Should we only include these when the program contains bit-vector operations, so we don't bloat the prelude when the program doesn't?

Reply:
Oh yeah one related point, we might need access to some none rust native bv operations. For example we might need bit selection, bv concatenation. We might want to group them into a separate module? Then if that module is imported, we can add these declarations into the prelude, and in the normal case it just won't be there. Thoughts?

Reply:
If you decide to implement this no-bloat optimization (no hurry), you can do it on a module-by-module basis, since Verus verifies one module at a time.  In this case, you could just move the code above to a different function in `prelude.rs` that gets called on demand for the modules that need it.